examples/2-handle-prompt/index.js

```diff
@@ -0,0 +1,100 @@
+import http from 'http';
+import { Octokit } from "@octokit/rest";
+import { 
+    createAckEvent, 
+    createDoneEvent, 
+    createTextEvent, 
+    parseRequestBody, 
+    prompt
+ } from "@copilot-extensions/preview-sdk";
+
+
+// Define the handler function
+const handler = async (request, response) => {
+    if (request.method === 'POST') {
```

Comment:
I would reverse condition and do an early return to avoid indentation

```suggestion
    if (request.method !== 'POST') {
```